src/visualizer.js

```diff
@@ -64,7 +64,7 @@ if (config.visualizer.enabled) {
           });
         }
         // draw creep positions
-        if (config.visualizer.showCreeps) {
+        if (config.visualizer.showCreeps && room.memory.position) {
```

Comment:
I just did `delete Memory.rooms` which could be the reason. But we should handle these cases. In the past there were also Memory issues on the server and we want to be able to recover.